SimPEG/regularization/base.py

```diff
@@ -634,6 +634,53 @@ def _multiplier_pair(self):
 
 
 class WeightedLeastSquares(ComboObjectiveFunction):
+    r"""Weighted least squares measure on model smallness and smoothness.
+
+    L2 regularization with both smallness and smoothness (first order
+    derivative) contributions.
+
+    Parameters
+    ----------
+    mesh : discretize.base.BaseMesh
```

Comment:
```suggestion
    mesh : discretize.base.BaseMesh
        The mesh on which the model parameters are defined. This is used for constructing difference operators for the smoothness terms. 
```

Reply:
just a suggestion, but might be helpful

Reply:
Yeah, more detail is better. The only reason I added it here was because it was already being used as an example of documentation in the style guide.